udata/core/pages/api.py

```diff
@@ -0,0 +1,46 @@
+from flask import request
+from flask_login import current_user
+
+from udata.api import API, api
+from udata.api_fields import patch
+
+from .models import Page
+
+ns = api.namespace("pages", "Pages related operations (beta)")
+
+common_doc = {"params": {"page": "The page ID or slug"}}
+
+
+@ns.route("/", endpoint="pages")
+class PagesAPI(API):
+    @api.secure
+    @api.doc("create_page", responses={400: "Validation error"})
+    @api.expect(Page.__write_fields__)
+    @api.marshal_with(Page.__read_fields__, code=201)
+    def post(self):
```

Comment:
I think a `get` route would be nice here to help us debugging.

Reply:
Do we want to restrict it to admins ? This way, we have time to work on `private` or other fields for the organizations customization

Reply:
Yes, we can restrict it to admins to get started.
Regarding the performance issues, we could tackle it using reference fields, ex `dataset_ref_fields` to return a minimal payload. At least, we should make sure not to return the dataset resources for example.

Reply:
Done in https://github.com/opendatateam/udata/pull/3387/commits/8fe49224321133ee8180a65e85cad9d032d2a372